src/components/Target.vue

```diff
@@ -68,27 +68,30 @@ export default {
     },
     hours() {
       let j = this.buslist.length;
-      if (this.buslist[this.i].type == "F") {
+      if (this.buslist[this.i].type === "F") {
         return 0;
       } else {
+        if(this.buslist[this.i].time.isNaN()) return 0 //prevent NaN text shown
```

Comment:
Would be better `if(this.buslist[this.i].time.isNaN())` to `if (this.buslist[this.i].time.isNaN())`.(indentation)